home/home2.1/src/Main.java

```diff
@@ -0,0 +1,92 @@
+import java.io.*;
+import java.util.*;
+
+class StatisticsDatabase {
+    private Map<String, Integer> stats;
+    private String filePath;
+
+    public StatisticsDatabase(String filePath) {
+        this.filePath = filePath;
+        this.stats = new HashMap<>();
+    }
+
+    public void load() {
+        try (Scanner scanner = new Scanner(new File(filePath))) {
+            while (scanner.hasNext()) {
+                String name = scanner.next();
+                int count = scanner.nextInt();
+                stats.put(name, count);
+            }
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void save() {
+        try (PrintWriter writer = new PrintWriter(filePath)) {
+            for (Map.Entry<String, Integer> entry : stats.entrySet()) {
+                writer.println(entry.getKey() + " " + entry.getValue());
+            }
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void clear() {
+        stats.clear();
+        saveToFile();
+        System.out.println("The history was deleted");
```

Comment:
Minor: in general it is better to handle business logic to higher level. in this case it should not be a responsibility of the class itself but of the code that will use its API